deployment/terraform/dev/pools.tf

```diff
@@ -100,31 +100,6 @@ module "batch_pool_d3_v2_landsat" {
   subnet_id = module.resources.batch_nodepool_subnet
 }
 
-module "batch_pool_d3_v3_modis" {
```

Comment:
Also removing unused batch pool that had been deleted from the batch account manually.